### ISSUE TITLE
npm 2.x installation is broken

### ISSUE THREAD

**Issue by hipertracker** [anodynos/uRequire#61]
HOW to use it if it even cannot be installed? `npm install` does not work  :(
```
$ npm install
npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.10.34
npm ERR! npm  v2.1.14
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package urequire does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer urequire-rc-coffee-script@0.1.2 wants urequire@~0.7.0
npm ERR! peerinvalid Peer grunt-urequire@0.7.1 wants urequire@^0.7.0
npm ERR! peerinvalid Peer urequire-rc-exec@0.1.1 wants urequire@~0.7.0
npm ERR! peerinvalid Peer urequire-rc-less@0.1.1 wants urequire@~0.7.0
npm ERR! peerinvalid Peer urequire-ab-specrunner@0.2.0 wants urequire@^0.7.0-beta.11
npm ERR! peerinvalid Peer urequire-rc-inject-version@0.1.4 wants urequire@~0.7.0
npm ERR! peerinvalid Peer urequire-rc-import-keys@0.1.2 wants urequire@~0.7.0
npm ERR! peerinvalid Peer urequire-ab-grunt-contrib-watch@0.0.1 wants urequire@^0.7.0-beta.11

```

npm-debug.log [attached](http://pastiebin.com/54ad03757a571)

**Comment by hipertracker** [comment]
It must be somehow related to the Node version. My OS X is using Node v2.x. My Ubuntu and Windows 10.beta boxes use  Node 1.x and I CAN install `urequire` there...

**Comment by anodynos** [comment]
Thank for the report @hipertracker - I am afraid I haven't tested with npm 2.x - I will check its compatible for the v0.7 release. Regards

**Comment by michaelcarter** [comment]
+1 for this issue.

**Comment by mattydoincode** [comment]
+1 this is super blocking! Is there an alternative way to get this setup?

**Comment by anodynos** [comment]
Its probably due to npm dependencies - perhaps you can change the "^" etc dependencies in `package.json` of urequire to absolute ones (of the latest versions published) and it should work ok.

I will release a fix by next week. Sorry for this, I am in a big transition phase in my life and have limited time.

**Comment by mattydoincode** [comment]
No stress Agelos, best of luck to you and thanks for the tips.

**Comment by anodynos** [comment]
Ok! Found the culprit http://blog.npmjs.org/post/98131109725/npm-2-0-0 & some time for open source - I ll hopefully release a fix this weekend & v7.0 soon (its mostly docs that need revising).

**Comment by anodynos** [comment]
The `npm@2.x` issue was actually in the deps of `urequire` (eg `urequire-rc-inject-version`). These are all now updated on npm & the latest version `0.7.0-beta.14` should work without problems.
